R/uvhydrograph-render.R

```diff
@@ -36,10 +36,12 @@ createPrimaryPlot <- function(data, month){
   if(anyDataExist(primaryData)){
 
     primaryInfo <- parseUVSupplemental(data, "primary", primaryData)
+    plotEndDate <- tail(primaryInfo$plotDates,1) + hours(23) + minutes(45)
```

Comment:
@mhines-usgs Yeah, this is the way I did it for UV Hydro. It feels kind of hacky to do it this way? If you can think of a better way, you should definitely tell me.